docs/src/main/asciidoc/spring-cloud-config-client.adoc

```diff
@@ -0,0 +1,140 @@
+    -Dpath="/greeting" \
+    -Dextensions="spring-cloud-config-client"
+cd spring-cloud-config-quickstart
+----
+
+This command generates a Maven project with a REST endpoint and imports the `spring-cloud-config-client` extension.
+
+== GreetingController
+
+The Quarkus maven plugin automatically generated a `GreetingResource` JAX-RS resource in the
+`src/main/java/org/acme/spring/cloud/config/client/GreetingResource.java` file that looks like:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return "hello";
+    }
+}
+----
+
+As we want to use configuration properties read from the Config Server, we will update the `GreetingResource` to inject the `message` property. The updated code will look like this:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @ConfigProperty(name = "message", defaultValue="hello default")
+    String message;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return message;
+    }
+}
+----
+
+== Configuring the application
+
+Quarkus provides various configuration knobs under the `quarkus.scccc` root. For the purposes of this guide, our Quarkus application is going to be configured in `application.properties` as follows:
+
+[source,properties]
+----
+# use the same name as the application name that was configured when standing up the Config Server
+quarkus.application.name=a-bootiful-client
+# enable retrieval of configuration from the Config Server - this is off by default
+quarkus.scccc.enabled=true
+# configure the URI where the Config Server listens to HTTP requests - this could have been left out since http://localhost:8888 is the default
+quarkus.scccc.uri=http://localhost:8888
+----
+
+== Package and run the application
+
+Run the application with: `./mvnw compile quarkus:dev`.
+Open your browser to http://localhost:8080/greeting.
+
+The result should be: `Hello world`.
+
+== Run the application as a native executable
+
+You can of course create a native image using the instructions of the link:building-native-image[Building a native executable guide].
+
+== More Spring guides
+
+Quarkus support has more Spring compatibility features. See the following guides for more details:
+
+* link:spring-di[Quarkus - Extension for Spring DI]
+* link:spring-data-jpa[Quarkus - Extension for Spring Data JPA]
+* link:spring-security[Quarkus - Extension for Spring Security]
```

Comment:
Should you add this guide in the other guides now that we have it?

Reply:
Very good idea!

Reply:
Done